packages/apify/src/actor.ts

```diff
@@ -179,6 +177,9 @@ export class Actor<Data extends Dictionary = Dictionary> {
             this.config.useStorageClient(options.storage);
         }
 
+        // Init the event manager the config uses
```

Comment:
this looks wrong, we need to use the `PlatformEventManager`, without it the platform events (from websockets) wont work at all.

Reply:
The if check above will make the config use the PlatformEventManager, no?

Reply:
Right, I am blind. But then I dont understand why did you have to change it to use the one from config?

Reply:
Or is the fix just about moving the `init` call and the rest is unrelated? i mean the changes from `this.eventManager` to `this.config.getEventManager()`

Reply:
Oh wait a sec, now I get it. We dont want to use the platform implementation locally, but we did.

Reply:
Yep, thats the reason!